### PR TITLE
Feature/snippet edit

### PR DESCRIPTION
**Pull request by vrael560**

### CODE REVIEW

**Comment by blimusiek** [comment]
@vrael560 `npm run lint` before push :P

File: client/js/components/snippets/snippet.jsx [hunk 1]
```diff
@@ -56,7 +78,47 @@ export default class Snippet extends React.Component {
     }
   }
 
-  render() {
+  _getEditableSnippet() {
```
Comment (blimusiek) [inline client/js/components/snippets/snippet.jsx]: I think in the future we should split it into separate components because it starts to look like a mess

**Comment by blimusiek** [comment]
but commits looks good for me.

**Comment by blimusiek** [comment]
:+1:

File: client/css/app.scss [hunk 2]
```diff
@@ -3,3 +3,4 @@
 @import "reset";
 @import "general";
 @import "comments";
+@import "https://fonts.googleapis.com/icon?family=Material+Icons"
```
Comment (dwoznicka) [inline client/css/app.scss]: i understand you don't use it, then we can remove this. the rest is fine :+1:

**Pull request merged**